Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabStrip.css

```diff
@@ -52,6 +52,18 @@
   background-color: var(--color-standard-background);
 }
 
+.tabStripTabWrapper .tabStripTab a span.anchorBody,
```

Comment:
I'm not 100% sure what the most consistent approach is but my gut feeling says this should go after all the `a` rules since it's nested inside `a`